### DISCUSSION TITLE
Enchantements customisées

### DISCUSSION THREAD

**Discussion by Steellgold** [LunastiaMC/SkyBlock#31] [Ideas]
Proposez des enchantements qui peuvent être sympa à avoir sur le serveur

**Comment by Steellgold** [comment]
## Jésus
**Appliquable sur:** Bottes
**Niveau maximum:** 3

Niveaux:
- Niveau **1**:
  - Permet de marcher 25 blocs conséqutif sur l'eau
  - Temps de rechargement: **1 minutes**
- Niveau **2**:
  - Permet de marcher 50 blocs conséqutif sur l'eau
  - Temps de rechargement: **30 secondes**
- Niveau **3**:
  - Permet de marcher 100 blocs conséqutif sur l'eau
  - Temps de rechargement: **10 secondes**

**Reply by Pralexkid** [reply]
Pas mal
